source/stylesheets/refills/_badges.scss

```diff
@@ -1,36 +1,20 @@
-$badge-font-size: 0.75em !default;
-$medium-gray: #999 !default;
-$alert-color: #fff6bf !default;
-$error-color: #fbe3e4 !default;
-$notice-color: #e5edf8 !default;
-$success-color: #e6efc2 !default;
+$small-font-size: 0.75em;
+$badges: (
+  default: #999,
```

Comment:
How come the `!default` is removed from this color definition? The idea was to be able to let Bitters override this one with the `$medium-gray` it defines, Bitters is used.